### DISCUSSION TITLE
A203 Carrier Board | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1744] [Q&A]
# A203 Carrier Board | Seeed Studio Wiki

A203 Carrier Board

https://wiki.seeedstudio.com/reComputer_A203_Flash_System/

**Comment by yalexx** [comment]
At IDRobots we are trying to flash ) ✗ sudo ./flash.sh jetson-nano-devkit-emmc mmcblk0p1 
 lets see if we can

**Comment by Aatmaj-Zephyr** [comment]
Here is how I managed to do the steps

First I tried using SDK manager. I selected the download now and install later. But there is nothis in /home/<username>/nvidia/nvidia_sdk/JetPack_<version num>_Linux_<board name>_TARGETS/Linux_for_Tegra
the directory nvdia_sdk is empty.

I clicked on install option and disconnected the board. and the directory with all its contents were made. I then replaced files and resumed/restarted the process. It worked fine.

Jetson Linux got installed this way but ethernet was not working.  But I was trying to install jetpack version 5.1 something (latest version available for xavier nx in SDK manager) which is incorrect because drivers are only available for 5.0.2 at max. (There might be some fix around the ethernet issue, but no idea how many other errors might pop up with this mis match)

Then I tried the second method this time using the correct exact same versions of jetpack. It worked, I was able to install the jetson linux, but the runtime environments. cuda, opencv etc were not installed.

Then I installed them using the jetson SDK manager and they were installed.

Before installing these components, I had to make space for them by shifting by boot from 16gb nvme to a 256 gb external nvme steps here - https://github.com/jetsonhacks/rootOnNVMe If you are getting error in jetson manager about lack of space then this is likely the fix.

Also opencv with cuda acceleration was not installed through the sdk maager. I had to build it ans install it through script provided here https://forums.developer.nvidia.com/t/best-way-to-install-opencv-with-cuda-on-jetpack-5-xavier-nx-opencv-for-tegra/222777

**Reply by nfs0619** [reply]
Thank you very much for sharing such detailed insights and solutions in the comments! Your contribution not only helps solve potential issues but also provides valuable references and support for other developers. Once again, thank you for your support and contribution. Your input is incredibly valuable to us!
